app/src/main/kotlin/eu/quiqua/gloomhaven/statstracker/features/stats/viewModel/StatsViewModel.kt

```diff
@@ -17,7 +20,42 @@ class StatsViewModel @Inject constructor(application: android.app.Application) :
     }
 
     @Inject
-    lateinit var model: String
+    lateinit var stats: Stats
 
-    fun getStats(): String = model
+    val mutableHp: MutableLiveData<Int> = MutableLiveData()
+    val mutableXp: MutableLiveData<Int> = MutableLiveData()
+
+    fun currentHp(): LiveData<Int> {
+        if (mutableHp.value == null) {
+            mutableHp.value = stats.hp
+        }
+        return mutableHp
+    }
+
+    fun increaseHp() {
+        stats.hp += 1
+        mutableHp.value = stats.hp
+    }
+
+    fun decreaseHp() {
+        stats.hp -= 1
+        mutableHp.value = stats.hp
+    }
+
+    fun currentXp(): LiveData<Int> {
```

Comment:
neither is this one

Reply:
Also used here:
https://github.com/quiqua/gloomhaven-stats-tracker/blob/680ab7c5003b7a6953d6aabc89540dfc81c0434b/app/src/main/res/layout/activity_stats.xml#L55